core/src/main/kotlin/no/strooped/view/screen/JoinGameScreen.kt

```diff
@@ -70,6 +77,12 @@ class JoinGameScreen(private val controller: StroopedController) : Screen {
         joinButton.onClick {
             controller.connectToGame(username = username.text, joinPin = pin.text)
         }
+
+        errorMessage?.let {
+            println("Found error message $it")
+            val errorMessage = Label(it, Vector2(joinButtonPosition.x, joinButtonPosition.y + 75), 200f, 2f)
```

Comment:
I just pasted in some random values here, because I only wanted to verify that it works